tests/unit-tests/margo-forward.c

```diff
@@ -106,6 +118,61 @@ static MunitResult test_forward(const MunitParameter params[],
     return MUNIT_FAIL;
 }
 
+static MunitResult test_get_name(const MunitParameter params[],
+                                 void*                data)
+{
+    (void)params;
+    (void)data;
+    hg_return_t hret[5] = {0,0,0,0,0};
```

Comment:
This array isn't big enough (needs to be bumped up to [6] to accommodate the `hret[5] = margo_addr_free(ctx->mid, addr);` assignment further down to avoid possible stack corruption).

Unrelated to the memory bug, this statement and others like it can be simplified to `hg_return_t hret[6] = {0};` to be slightly cleaner.  C recognizes that syntax to zero out an entire array or struct without individually specifying each element.

Reply:
Thanks, I corrected and will merge the PR.